app/controllers/api/articles_controller.rb

```diff
@@ -49,4 +49,20 @@ def show
       message: 'Article not found.'
     }, status: 404
   end
+
+  def create
+    current_admin.articles.create(
+      title: params[:title], teaser: params[:teaser], body: params[:body],
+      article_type: params[:article_type], location: params[:location], category: params[:category]
+    )
```

Comment:
This could've been done much easier by permitting the params that an article should have.